Lint-Whoppers/nutshell
# created file stucture

# Description

Michelle created file structure for application

Fixes https://github.com/Lint-Whoppers/nutshell/issues/55

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1) `git fetch --all`
2) `git checkout mj-filestructure`
3) Check to see if new file structure is there (new JS files in Events, News, Friends, etc folders, and new empty JSON files in API folder)



# Checklist:
